src/components/GlobalFilter/globalFilterApi.ts

```diff
@@ -137,15 +137,10 @@ export const flatTags = memoize(
           }`;
         })
     );
-    return format
-      ? [
-          Workloads,
-          Object.entries<any>(SID || {})
-            .filter(([, { isSelected }]: [unknown, GroupItem]) => isSelected)
-            .reduce<any>((acc, [key]) => [...acc, key], []),
-          mappedTags,
-        ]
-      : mappedTags;
+    const sidArray = Object.entries<any>(SID || {})
+      .filter(([, { isSelected }]: [unknown, GroupItem]) => Boolean(isSelected))
```

Comment:
Consistency nit. Above on line 129 you use `===` and here you are using the `Boolean` func. Use one or the other in both places..